Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title> Girl Develop it</title>
+
+        <!--Stylesheets-->
+        <link rel="stylesheet" href="css/bootstrap.css">
+        <link rel="stylesheet" href="css/bootstrap-grid.min.css">
+        <link rel="stylesheet" href="css/style.css">
+
+        <!--Imported fonts-->
+        <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Oswald" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css?family=Josefin+Sans" rel="stylesheet">
+        
+        <script src="js/bootstrap.js"></script>
+    </head>
+    <body>
+        <div class="text-center main-header">
+            <img src="images/mainlogoWhite.png" class ="img-fluid" alt="Don't be shy. Develop it">
+        </div>
+
+        <h1 class="text-center"><u>About</u></h1>
+        <p class="text-center text-box">It can be intimidating for woman to learn and ask questions when they are in an <b>extreme minority.</b> While open and welcoming, today's budding developer community is up to <b>91% male.</b>  If we can empower more females with the confidence in their technological capabilities we can begin to change this landscape</p>
+
+        <br>
+        
+        <div class="containter pink-elements">
+            <h1 class="text-center"><u>Our Locations</u></h1>
+            <br>
+            <h2 class="text-center"><i>United States</i></h2>
+            <div class="row">
+                <div class="col-md-6">
+                    <h2 class="text-center">New York</h2>
+                    <img src="/images/newYork.jpg" alt="New York" class="img-thumbnail img-fluid">
+                </div>
+                <div class="col-md-6">
+                    <h2 class="text-center">Columbus</h2>
+                    <img src="/images/columbus.jpg" alt="Columbus" class="img-thumbnail img-fluid">
+                </div>
+            </div>
+            <br>
+            <div class="row">
+                <div class="col-md-6">
+                    <h2 class="text-center">Austin</h2>
+                    <img src="/images/austin.jpg" alt="Austin" class="img-thumbnail img-fluid">
```

Comment:
If you start your relative path with a "/", you are referring to the root of the web. This is not recommended, unless you are 100% that your root is going to be the folder where all your files are stored. For example, in my computer, it's looking for the file at this path: `file:///images/newYork.jpg`, which is the very root of my computer.

If you remove the initial "/", you will refer to the current folder, that might be a solution.

But, I recommend to use "./" whenever you can. The "." means current folder, so that show other developers that you are explicitly referring to current folder, so your routes will look like: "./images/austin.jpg"